packages/create-spectacle/tsconfig.json

```diff
@@ -4,6 +4,8 @@
     "moduleResolution": "node",
     "module": "commonjs",
     "outDir": "./bin",
+    // https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-14
+    "target": "ES2020",
```

Comment:
@carlos-kelly I've updated the spectacle bin output to be modern supported node target (14)